Formula/h/homebank.rb

```diff
@@ -8,8 +8,8 @@ class Homebank < Formula
   license "GPL-2.0-or-later"
 
   livecheck do
-    url "http://homebank.free.fr/public/sources/"
-    regex(/href=.*?homebank[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    url "https://deb.debian.org/debian/pool/main/h/homebank/"
+    regex(/href=.*?homebank[._-]v?(\d+(?:\.\d+)+)\.o/i)
```

Comment:
```suggestion
    regex(/href=.*?homebank[._-]v?(\d+(?:\.\d+)+)\.orig\.t/i)
```

As mentioned in another PR, we only truncate tarball extensions (i.e., `\.t`) in regexes. In this case, `.orig` is part of the suffix, so this should use `\.orig\.t`, as in other Debian `livecheck` block regexes.